Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -162,6 +209,55 @@ public Task<RecurringWebinar> CreateRecurringWebinarAsync(string userId, string
 				.AsObject<RecurringWebinar>();
 		}
 
+		/// <summary>
+		/// Updates an existing recurring webinar for a user.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="recurrence">Recurrence information.</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// Zoom will return an HTTP 200 response when there is no webinar subscription present.
+		/// A successful update is reported by an HTTP 204 value.
+		/// </returns>
+		public Task UpdateRecurringWebinarAsync(long webinarId, string topic, string agenda, DateTime? start, int duration, RecurrenceInfo recurrence, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default)
+		{
+			var data = new JObject()
+			{
+				{ "type", start.HasValue ? 9 : 6 }
```

Comment:
This logic works when creating a new webinar but not when updating an existing one. Omitting a value (such as the start time for example) when invoking the UpdateWebinar method means that you want to preserve the current value. If we leave this logic, it means that the webinar type will get changed when developer omit the start time when invoking the UpdateWebinar method which is not the desired outcome.

Reply:
Removing in next PR